### ISSUE TITLE
Minter script error

### ISSUE THREAD

**Issue by comtechnet** [onflow/kitty-items#83]
The script ./setup-minter.sh results in cadence errors. Env vars are passed and values were verified. 
Server is running and postgres instance/container is running.

Output
```
david@ttglt001 MINGW64 ~/kitty-items/api (master)
$ ./setup-minter.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    73  100    73    0     0      7      0  0:00:10  0:00:09  0:00:01    17{"transaction":{"status":4,"statusCode":0,"errorMessage":"","events":[]}}
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    73  100    73    0     0      7      0  0:00:10  0:00:09  0:00:01    17{"transaction":{"status":4,"statusCode":0,"errorMessage":"","events":[]}}
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    75  100    75    0     0      7      0  0:00:10  0:00:09  0:00:01    17{"transactionId":{"status":4,"statusCode":0,"errorMessage":"","events":[]}}
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   547  100   502  100    45     67      6  0:00:07  0:00:07 --:--:--   158<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>[Error Code: 1101] cadence runtime error Execution failed:<br>error: invalid argument at index 0: decodeing argument failed: [Error Code: 1052] transaction arguments are invalid: (argument is not json decodable: failed to decode value: runtime error: slice bounds out of range [:2] with length 0)<br>--&gt; 56f938211cac921dccf4830a949510b19cab90521e610b8ad087351da91edd44<br></pre>
</body>
</html>

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   544  100   502  100    42     78      6  0:00:07  0:00:06  0:00:01   119<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>[Error Code: 1101] cadence runtime error Execution failed:<br>error: invalid argument at index 0: decodeing argument failed: [Error Code: 1052] transaction arguments are invalid: (argument is not json decodable: failed to decode value: runtime error: slice bounds out of range [:2] with length 0)<br>--&gt; f15bae6de32c274dac7cfef583a8bfc84ed0212cad616981dfa2849c058525b4<br></pre>
</body>
</html>

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   834  100   795  100    39    123      6  0:00:06  0:00:06 --:--:--   247<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>[Error Code: 1101] cadence runtime error Execution failed:<br>error: unexpectedly found nil while forcing an Optional value<br> &nbsp;--&gt; c9248a39fe523324959c2f346f067ffad37cc8d8e8befc6575367f436c8aa646:34:83<br> &nbsp; |<br>34 | &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br> &nbsp; | &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;^<br></pre>
</body>
</html>
```

**Comment by sideninja** [comment]
Can you please confirm all the previous steps were executed successfully and the Minter account was created?

**Comment by KesleyDavid** [comment]
same problem for me.
all previous steps have been successfully performed

**Comment by 10thfloor** [comment]
Closed by #91
